### PR TITLE
Update wix translations

### PR DESCRIPTION
**Pull request by sblaisot**
- update translation source to remove LangID
- update translations from transifex
- add translations update instructions into README-Translations.md

### CODE REVIEW

**Comment by rryan** [comment]
thanks

File: build/wix/Localization/README-Translations.md [hunk 1]
```diff
@@ -2,3 +2,26 @@
...
+```
+$ cd build/wix/Localization
+build/wix/Localization$ for i in po/*.po; do destfile=${i#po/}; /path/to/po2wxl.py -l Language -f -p 60 $i mixxx_${destfile%.po}.wxl; done
+```
+
+explainations for the above line :
+For earch po file in the po subdirectory, call po2wxl to transform it to wxl file with the following options :  
+`-l Language` add a string with Id Language containing the auto-determined LCID  
+`-f` Overwrite the wxl file if it already exists
+`-p 60` don't transform po files if less than 60% of strings are translated
```
Comment (esbrandt) [inline build/wix/Localization/README-Translations.md]: FWIW You can also use the transifex client for this kind of operation
``tx pull --minimum_perc=60``
Personally, i'd drop translations below 100%. It are less than 30 strings for the installer.

I`d do it for the main app too, with a lower threshold, but nothing has materialized from the discussion back than @https://bugs.launchpad.net/mixxx/+bug/1298977

**Comment by sblaisot** [comment]
thanks @esbrandt for the tip on the transifex client. I will update the README the next time I will update translations.

There is really far more than the 25 strings you have in transifex. There are dozens of them that are standard messages with the translations built-in in Wix. The 25 strings we need a translation for are our own custom error messages, the components and the product name (but, hey, "Mixxx" is understandable in any language).

So I prefer to have a 60% translation (some rare error messages are not translated, but the majority of the interface is) rather than no translation at all.

**Comment by sblaisot** [comment]
I just made a count and found 553 built-in translated strings.
so if our translation is done at 60%, the whole installer is translated at 98% ;)

**Pull request merged**